docs-rtd/howto/manage-service-accounts.md

```diff
@@ -5,24 +5,6 @@
 In the Juju ecosystem, service accounts are supported only when using [JAAS](https://canonical-jaas-documentation.readthedocs-hosted.com/en/latest/).
 ```
 
-
-(manage-access-to-a-service account)=
-## Manage access to a service account
-
-When using Juju with JAAS, to grant access to a Juju controller connected to JIMM, in your Terraform plan add a resource type `juju_jaas_access_controller`. Access can be granted to one or more users, service accounts, roles, and/or groups. You must specify the model UUID, the JAAS controller access level, and the list of desired users, service accounts, roles, and/or groups. For example:
```

Comment:
Looks like this doc was wrong in the first place. The heading says "access to a service account" and then it talks about managing access to the controller.